File: Objects/Tour.java
package Objects;

import java.util.ArrayList;

public class Tour {
    private static int n = 0;
    private final int ID;
    private final String guide;
    private ArrayList<String> visitors = new ArrayList<String>();
    private final int money;
    private final String date;
    private final int kidsAmount;
    private final int adultsAmount;

    public Tour(String guide, ArrayList<String> visitors, int money, String date, int kidsAmount, int adultsAmount) {
        this.guide = guide;
        this.visitors = visitors;
        this.money = money;
        this.date = date;
        this.kidsAmount = kidsAmount;
        this.adultsAmount = adultsAmount;
        n++;
        ID = n;
    }

    public String getDetails(){
        String info = String.format("\nTour number: #%s%nGuide in charge: %s, Visitors: ",ID,guide);
        for (String visitor : visitors) {
            info = info + visitor + ". ";
        }
        info = info+String.format("%nTotal in tickets: $%d, date: %s, number of kids: %d & number of adults: %d",money,date,kidsAmount,adultsAmount);
        return info;
    }
}

File: Tour.py
class Tour:
    __guide = " "
    __visitors = []
    __money = 0
    __date = " "
    __kids_amount = 0
    __adults_amount = 0

    def __init__(self, guide, visitors, money, date, kids_amount, adults_amount):
        self.__guide = guide
        self.__visitors = visitors
        self.__money = money
        self.__date = date
        self.__kids_amount = kids_amount
        self.__adults_amount = adults_amount

    def get_details(self):
        info = "\nGuide in charge: " + self.__guide + ". Visitors: "
        i = 0
        for i in self.__visitors:
            kk = " " + self.__visitors[i] + ", "
            i = i + 1
        info = info + kk + "\nTotal in tickets: $" + str(self.__money) + ", date: " + self.__date + ", number of kids: " + str(self.__kids_amount) + ", number of adults: " + str(self.__adults_amount)
        return info
